pkg/txn/rpc/server.go

```diff
@@ -80,8 +81,13 @@ func NewTxnServer(
 		opt(s)
 	}
 
+	mp, err := mpool.NewMPool("txn-server-rpc", 0, mpool.NoFixed)
```

Comment:
dont you want mpool to have some fixed pre allocated buffer?

Reply:
Not required, unable to determine the need for a fixed resident memory size